massdash/peakPickers/pyMRMTransitionGroupPicker.py

```diff
@@ -134,6 +134,6 @@ def pick(self, transitionGroup: TransitionGroup) -> List[TransitionGroupFeature]
         # Calculate the consensus boundaries and integrated intensity
         transitionGroupFeatures = []
         for idx, row in newPeaks.iterrows():
-            transitionGroupFeatures.append(TransitionGroupFeature(row['leftBoundary'], row['rightBoundary'], areaIntensity=row['areaIntensity'], consensusApexIntensity=row['apexIntensity']))
+            transitionGroupFeatures.append(TransitionGroupFeature(row['leftBoundary'], row['rightBoundary'], areaIntensity=row['areaIntensity'], consensusApexIntensity=row['apexIntensity'], consensusApex=row['peakApex']))
```

Comment:
Whats the difference between consensusApexIntensity and consensusApex (is this for the consensus RT across runs or precursors)?

Reply:
consensusApexIntensity is the apex intensity value (across all transitions MS2 level) consensusApex is the retention time value of this point